specs/protocol/safe-extensions.md

```diff
@@ -116,46 +112,6 @@ The following security properties must be upheld by the `DeputyGuardianModule`:
 1. The module must format calldata correctly such that the target it calls performs the expected
    action.
 
-## Deputy Guardian Safe
-
-### Deputy Pause Module
-
-The Deputy Guardian Safe (currently the Optimism Foundation Safe) utilizes the Deputy Pause Module
-to remove the need for brittle pre-signed pause transactions and to speed up the reaction speed of
-the Deputy Guardian Safe for the specific purpose of triggering the Superchain-wide pause action.
-The Deputy Pause Module is explicitly designed to be used with an Externally Owned Account and is
-not designed to function with a smart contract deputy.
-
-#### Invariants
-
-1. Must enforce that the Deputy account is an EOA.
```

Comment:
this invariaht did not make it into the new document, I see no mention of 'EOA' there.

Reply:
Fixed